spec/mailers/user_mailer_spec.rb

```diff
@@ -1,6 +1,114 @@
 require "spec_helper"
 
 describe UserMailer do
+  describe "creatorship invitation" do
```

Comment:
Could you include the underscore here, as you've done with the others?

Reply:
Fixed